### DISCUSSION TITLE
How can I set the NameTag of a Player and how can I sort the Tablist?

### DISCUSSION THREAD

**Discussion by FloGamerMC** [Minestom/Minestom#1055] [Help]
I have 2 questions, the first how can I set the NameTag of a Player and the second how can I sort the Tablist?

**Comment by KrystilizeNevaDies** [comment]
We do not have a proper tablist api yet, so you'd have to use packets or pr an api.

**Reply by FloGamerMC** [reply]
and how can I set the NameTag of a Player
